src/Transport/Seam/MessagePump.cs

```diff
@@ -98,7 +97,7 @@ public void Start(PushRuntimeSettings limitations)
         /// </summary>
         IOperateTopologyInternal DetermineTopologyOperator(string pushSettingsInputQueue)
         {
-            if (satelliteTransportAddresses.Contains(pushSettingsInputQueue))
+            if (pushSettingsInputQueue != localAddress)
```

Comment:
Could this change be described as:

> instead of searching for the address in a set of satellites, now it checks is it different from the local address. If yes, it creates a new operator. The condition is more inclusive, works for all the previous cases and potentially covers some new cases?